src/VolumetricFlow.elm

```diff
@@ -0,0 +1,164 @@
+module VolumetricFlow exposing
+    ( VolumetricFlow, CubicMetersPerSecond
+    , cubicMetersPerSecond, inCubicMetersPerSecond
+    , litersPerSecond, inLitersPerSecond, cubicCentimetersPerSecond, inCubicCentimetersPerSecond
+    , cubicFeetPerSecond, inCubicFeetPerSecond, imperialGallonsPerMinute, inImperialGallonsPerMinute
+    , usLiquidGallonsPerMinute, inUsLiquidGallonsPerMinute, usDryGallonsPerMinute, inUsDryGallonsPerMinute
+    )
+
+{-| A `VolumetricFlow` represents a flow rate of volume per unit time measured
+in cubic meters per second, cubic feet per second, liters per second, US liquid
+gallons per minute, etc. It is stored as a number of cubic meters per second.
+
+Note that since `CubicMetersPerSecond` is defined as `Rate CubicMeters Seconds`,
+you can construct `VolumetricFlow` values using `Quantity.per`:
+
+    volumetricFlow =
+        volume |> Quantity.per duration
+
+@docs VolumetricFlow, CubicMetersPerSecond
+
+
+## Metric
+
+@docs cubicMetersPerSecond, inCubicMetersPerSecond
+@docs litersPerSecond, inLitersPerSecond, cubicCentimetersPerSecond, inCubicCentimetersPerSecond
+
+
+## Imperial
+
+@docs cubicFeetPerSecond, inCubicFeetPerSecond, imperialGallonsPerMinute, inImperialGallonsPerMinute
+
+
+## US Custom
+
+@docs usLiquidGallonsPerMinute, inUsLiquidGallonsPerMinute, usDryGallonsPerMinute, inUsDryGallonsPerMinute
+
+-}
+
+import Constants
+import Duration exposing (Seconds)
+import Quantity exposing (Quantity(..), Rate)
+import Volume exposing (CubicMeters)
+
+
+{-| -}
+type alias CubicMetersPerSecond =
+    Rate CubicMeters Seconds
+
+
+{-| -}
+type alias VolumetricFlow =
+    Quantity Float CubicMetersPerSecond
+
+
+
+----- CONSTANTS -----
+
+
+oneUsLiquidGallonPerMinute : Float
+oneUsLiquidGallonPerMinute =
+    Constants.usLiquidGallon / Constants.minute
+
+
+
+----- CONVERSIONS -----
+
+
+{-| Construct a volumetric flow from a number of cubic meters per second.
+-}
+cubicMetersPerSecond : Float -> VolumetricFlow
+cubicMetersPerSecond numCubicMetersPerSecond =
+    Quantity numCubicMetersPerSecond
+
+
+{-| Convert a volumetric flow to a number of cubic meters per second.
+-}
+inCubicMetersPerSecond : VolumetricFlow -> Float
+inCubicMetersPerSecond (Quantity numCubicMetersPerSecond) =
+    numCubicMetersPerSecond
+
+
+{-| Construct a volumetric flow from a number of liters per second.
+-}
+litersPerSecond : Float -> VolumetricFlow
+litersPerSecond numLitersPerSecond =
+    cubicMetersPerSecond (Constants.liter * numLitersPerSecond)
+
+
+{-| Convert a volumetric flow to a number of liters per second.
+-}
+inLitersPerSecond : VolumetricFlow -> Float
+inLitersPerSecond volumetricFlow =
+    inCubicMetersPerSecond volumetricFlow / Constants.liter
+
+
+{-| Construct a volumetric flow from a number of cubic centimeters per second.
+-}
+cubicCentimetersPerSecond : Float -> VolumetricFlow
```

Comment:
This should actually be cubic centimeters per _minute_ - according to https://en.wikipedia.org/wiki/Volumetric_flow_rate, "standard cubic centimeters per minute" is a common unit of flow rate.

Also, careful with conversion factors - here the `1.0e-2` should actually be cubed to get the correct conversion factor, since you're dealing with cubic centimeters! This should probably be updated to use the constants approach as discussed.